test/integration/addons_test.go

```diff
@@ -41,6 +41,11 @@ import (
 
 // TestAddons tests addons that require no special environment -- in parallel
 func TestAddons(t *testing.T) {
+
+	if !Arm64Platform() {
+		t.Skipf("Skip helm addon test for arm64")
```

Comment:
Can you add a GitHub issue to track fixing this? That way you can put the URL in the skip message.

Reply:
https://github.com/kubernetes/minikube/issues/10144

Reply:
lets add the issue in the Skip Message.

simmilar to other Skips, lets add a github issue and Write the Issue in the Skip Message